SpokestackTray/Models/TrayConfiguration.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  TrayConfiguration.swift
+//  SpokestackTray
+//
+//  Created by Cory D. Wiles on 8/23/20.
+//
+
+import Foundation
+import UIKit
+import Spokestack
+
+open class TrayConfiguration {
+    
+    // MARK: Public (properites)
+
+    open var buttonWidth: CGFloat = 60.0
+    
+    open var sayGreeting: Bool = true
+    
+    open var greeting: String = "Welcome to Spokestack"
+    
+    open var cliendId: String = ""
```

Comment:
I think it's good to include the publicly-distributed spokestack id/secret as the defaults here. See the framework's `SpeechConfiguration` for the values.